.env.sample

```diff
@@ -0,0 +1,5 @@
+LOCALHOST_API_URL=https://localhost:8080
+LOCALHOST_REDIRECT_URL=https://localhost:8080/auth/callback
+FOUNDRY_CLIENT_ID=<Fill in Client ID for your application>
+FOUNDRY_ONTOLOGY_API_URL=<Fill in your Foundry instance base URL, e.g. https://example.palantir.com>
```

Comment:
Current
```
FOUNDRY_ONTOLOGY_API_URL <-> LOCALHOST_API_URL
PRODUCTION_REDIRECT_URL <-> LOCALHOST_REDIRECT_URL
```

Should we try and make these follow more similar naming so it's obvious which settings are used in production?

Suggestion 1
```
PRODUCTION_API_URL <-> LOCALHOST_API_URL
PRODUCTION_REDIRECT_URL <-> LOCALHOST_REDIRECT_URL
```

Suggestion 2
```
PRODUCTION_API_URL <-> DEVELOPMENT_API_URL
PRODUCTION_REDIRECT_URL <-> DEVELOPMENT_REDIRECT_URL
```

Alternatively, rather than setting up different environment variables we could consider setting up different .env files e.g. `.env.development`/`.env.production` so it's simply `API_URL`/`REDIRECT_URL`. With the current setup using dotenv that could be achieved [setting the path](https://stackoverflow.com/a/55426597) based on the NODE_ENV variable.

Reply:
also `FOUNDRY_ONTOLOGY_API_URL` does not get swapped out, since it's needed to be a fixed value for webpack.config.js